libraries/ESP8266WebServer/src/ESP8266WebServer-impl.h

```diff
@@ -35,6 +35,12 @@
 #define DEBUG_OUTPUT Serial
 #endif
 
+#ifdef DEBUG_ESP_HTTP_SERVER
```

Comment:
Could you convert the other `DEBUG_OUTPUT.printf()`s to your new macro as a bit of cleanup?